__tests__/get-release-names.test.ts

```diff
@@ -0,0 +1,120 @@
+import {getReleaseNames} from '../src/get-release-names'
```

Comment:
no spaces here? `{ getReleaseNames }`

Reply:
🤷‍♀️ This project came with its own Prettier config and I didn't feel like changing it